checker/src/org/checkerframework/checker/nullness/NullnessVisitor.java

```diff
@@ -60,7 +60,7 @@
 public class NullnessVisitor extends InitializationVisitor<NullnessAnnotatedTypeFactory,
         NullnessValue, NullnessStore> {
     // Error message keys
-    private static final /*@CompilerMessageKey*/ String ASSIGNMENT_TYPE_INCOMPATIBLE = "assignment.type.incompatible";
+    // private static final /*@CompilerMessageKey*/ String ASSIGNMENT_TYPE_INCOMPATIBLE = "assignment.type.incompatible";
```

Comment:
It was an unused field and I commented it out to get rid of the warning. I kept it in case a use case arises.